src/App.js

```diff
@@ -36,27 +36,29 @@ function App() {
       ['', '', '', '', ''],
     ],
     cellStatuses: () => Array(6).fill(Array(5).fill(status.unguessed)),
-    currentRow: 0,
-    currentCol: 0,
+    currentRow: () => 0,
```

Comment:
what's the advantage of wrapping these all in functions?

Reply:
consistency. some of them already were functions, for sensible reasons (e.g.: random-answer, since it has to change every time), and some were functions for less clear reasons (e.g.: letters-guessed and cell-statuses, perhaps defensively, to prevent updating init structure in place?).

Reply:
I think the only ones we need to have in a function are `answer` and `letterStatuses`

Reply:
ok, i removed it for the others

Reply:
ok, i removed it for the others